docs/integration/java/RuoYi.md

```diff
@@ -75,33 +75,31 @@ public R<?> callback(@RequestBody CodeBody code) {//we should define a CodeBody
 ```
 In this method, we use casdoor-SpringBoot-sdk method and slightly modified RuoYi-Cloud Method.
 
-For exmaple RuoYi-Cloud original register with account and password, i change the register with account like casdoorRegister.
+For example, RuoYi-Cloud original register with account and password, I change the register with account like casdoorRegister.
 
 I also add a method to execute whether this account exists like getUserByCasdoorName and change execute userinfo with account and password to with account.
 
-It's eazy, because we only need delete the part of checking password.
-## Step5. Summary
+It's easy, because we only need to delete the part of checking password.
+## Step 5. Summary
 ### 5.1 front-end 
-- We need delete original login and register.
-- We also need accept code and state and send them to back-end.
+- We need to delete original login and register.
+- We also need to accept code and state and send them to back-end.
 ### 5.2 back-end
-RuoYi back-end have perfect login and register function. We just need change a little, so it is very convenient.
-## Step6. Detailed steps
-1. Deploy and configure casdoor.We must take care of organization's password type which should choose bcrypt because
-RuoYi-Cloud's password type is bcrypt.
-2. We should use casdoor syncers to copy database user to your casdoor organization. This step can make the original account
-import to casdoor.
+RuoYi back-end has perfect login and registration function. We just need to change a little, so it is very convenient.
+## Step 6. Detailed steps
+1. Deploy and configure casdoor. We must take care of the organization's password type which should choose bcrypt because RuoYi-Cloud's password type is bcrypt.
+2. We should use casdoor syncers to copy database users to your casdoor organization. This step can make the original account import to casdoor.
 3. After we deployed casdoor, we should change front-end. We should close RuoYi check code 
 ![checkcode Switch](/img/RuoYi-Cloud_loginSwitch.png)
 
  Note that RuoYi-Cloud checkcode needs close in nacos again.
- Note that RuoYi-Cloud open register function need change sys.account.registerUser to true. 
+ Note that if you want to register RuoYi-Cloud, you need to ensure the authenticity of your account and user name.
```

Comment:
I don't think it's a grammar error.
`sys.account.registerUser` is a option, open registration function requires this option to be turned on.

Reply:
OK, I got it! I'll revise it in my next commit.

Reply:
OK, I got it! I'll revise it in my next submission.